sup3r/cli/base.py

```diff
@@ -0,0 +1,211 @@
+
+# -*- coding: utf-8 -*-
+"""
+Sup3r base CLI class.
+"""
+import click
+import logging
+import os
+
+from reV.pipeline.status import Status
+
+from rex.utilities.execution import SubprocessManager
+from rex.utilities.hpc import SLURM
+from rex.utilities.loggers import init_mult
+
+from sup3r.version import __version__
+from sup3r.pipeline.config import BaseConfig
+from sup3r.utilities import ModuleName
+
+
+logger = logging.getLogger(__name__)
+
+
+class BaseCLI:
+    """Base CLI class used to create CLI for modules in ModuleName"""
+
+    @classmethod
+    def from_config(cls, module_name, module_class, ctx, config_file, verbose):
```

Comment:
What all is preventing this from being used everywhere? The multi-node jobs? Could the first part through line ~83 be put into a "get_from_config_parameters()" method that could be used everywhere? 

I'm assuming this first part is basically copied from cli to cli but the multi-node submission is unique in fwp, solar, and bias.

Reply:
Yeah data-extract and data-collect are unique as well but good call on extracting the first bit.